erts/doc/src/erlang.xml

```diff
@@ -52,6 +59,300 @@
   </description>
 
   <datatypes>
+    <datatype_title>Predefined datatypes</datatype_title>
+    <datatype>
+      <name name="any" n_vars="0"/>
+      <desc>
+        <p>The top type, the set of all Erlang terms.</p>
```

Comment:
I think we should avoid **top** and **bottom**; I don't think they are concepts generally known.

```suggestion
        <p>All possible Erlang terms. Synonym for <c>term()</>.</p>
```